challenges/index.html

```diff
@@ -1,61 +1,60 @@
 <!DOCTYPE html>
```

Comment:
Don't change this file, at least not in the challenges' PRs.
You can do it later in your main branch.